src/components/Settings.vue

```diff
@@ -320,6 +320,26 @@
                           persistent-hint
                         ></v-switch>
                       </v-flex>
+                      <v-flex xs6 v-if="gateway.hassDiscovery">
+                        <v-text-field
+                          v-on="on"
+                          v-model="gateway.entityTemplate"
+                          label="Entity name template"
+                          persistent-hint
+                          hint="Template which generates entity names"
+                        ></v-text-field>
+                      </v-flex>
+                      <v-flex xs6 v-if="gateway.hassDiscovery">
+                        <div>
+                          Default: <code>%loc-%n_%ob</code> <br />-
+                          <code>%n</code>: Node Name <br />- <code>%loc</code>:
```

Comment:
Maybe we could also add the node id as placeholder `%nid`

Reply:
i agree! good one! will do

Reply:
changing also the node placeholder to %nn! as this fits on the scenario of conflict